tests/regression_tests/deplete_with_transfer_rates/test.py

```diff
@@ -82,5 +82,5 @@ def test_transfer_rates(run_in_tmpdir, model, rate, dest_mat, power, ref_result)
     res_ref = openmc.deplete.Results(path_reference)
     res_test = openmc.deplete.Results(path_test)
 
-    assert_atoms_equal(res_ref, res_test, 1e-4)
-    assert_reaction_rates_equal(res_ref, res_test)
+    assert_atoms_equal(res_ref, res_test, 1e-3)
+    assert_reaction_rates_equal(res_ref, res_test, 1e-1)
```

Comment:
The reaction rate relative tolerance of 1e-1 up from 1e-5 feels not great. Is this because the updated files came from local, not the CI machine? Which I'm not sure we could easily pull if we wanted too

Reply:
I agree. I do want to get to the bottom of it and I think we need the tests to be able to pass locally so pulling results from CI servers isn't ideal either.

Reply:
@drewejohnson I got to the bottom of it (or at least close to the bottom). See #2779 for a bit of an explanation. Once that one is merged, I'll remove all the test changes in this PR.